creator-node/src/routes/tracks.js

```diff
@@ -37,9 +44,35 @@ const SaveFileToIPFSConcurrencyLimit = 10
 
 module.exports = function (app) {
   /**
-   * upload track segment files and make avail - will later be associated with Audius track
-   * @dev - Prune upload artifacts after successful and failed uploads. Make call without awaiting, and let async queue clean up.
+   * Add a track transcode task into the worker queue. If the track file is uploaded properly (not transcoded), return successResponse
+   * @note this track content route is used in conjunction with the polling.
    */
+  app.post('/polling_track_content', authMiddleware, ensurePrimaryMiddleware, ensureStorageMiddleware, syncLockMiddleware, handleTrackContentUpload, handleResponse(async (req, res) => {
```

Comment:
tbh, i kinda disagree w/ @dmanjunath  on this. having it as the same route w/ a flag will make the migration path a bit cleaner -- because we can change the default flag value, etc. over time, but no strong conviction. just thought i'd call out. dealer's choice.

maybe /track_content_async or something like that though since this isn't actually doing any polling

Reply:
yeah, I think what we have is ok. it's just a little more of a cleanup. I would prefer we left it as is and remove later